packages/web/pages/user/my-account/bookmarks.js

```diff
@@ -70,11 +85,33 @@ const MyBookmarks = ({
 					<MainContent>
 						{bookmarks.length ? (
 							<DataGrid
-								data={bookmarks.map(({ id, title, status, slug }) => ({
+								data={bookmarks.map(({ id, title, status }) => ({
```

Comment:
You removed `slug` from table header. This way we can't click in row to open technology details page. Any reason for doing that?

![image](https://user-images.githubusercontent.com/13091635/102814966-538a3580-43aa-11eb-83f5-2fcc18036075.png)

This column is hidden because of `hideItemsByKey={['slug']}` prop, so don't worry.

Reply:
No. My mistake 😬